packages/core/src/extensionCommon.ts

```diff
@@ -81,6 +81,7 @@ export async function activateCommon(
     await initializeComputeRegion()
 
     globals.contextPrefix = '' //todo: disconnect supplied argument
+    globals.isEmbedded = !!(vscode.window.sendMessage && vscode.window.receiveMessage)
```

Comment:
This should live in `initialize` https://github.com/aws/aws-toolkit-vscode/blob/2ef5d5e888fe634d2e04d0d1d2238b99a2b1b824/packages/core/src/shared/extensionGlobals.ts#L139